nemo/collections/asr/models/label_models.py

```diff
@@ -337,6 +338,23 @@ def setup_finetune_model(self, model_config: DictConfig):
 
         logging.info(f"Changed decoder output to # {self.decoder._num_classes} classes.")
 
+    @torch.no_grad()
+    def get_embedding(self, path2audio_file):
+        audio, sr = librosa.load(path2audio_file, sr=None)
+        if sr != 16000:
+            raise ValueError("Make sure input audio has sample rate of 16Khz")
```

Comment:
Why are we throwing an error here? Shouldn't we just resample the audio to 16 kHz? That is the current behavior in ASR.

Reply:
we could do that as well

Reply:
Resampling it is better.